lib/content/gittenify.js

```diff
@@ -0,0 +1,36 @@
+import catNames from 'cat-names';
+import mutationSummary from 'mutation-summary';
+import catPhoto from './catPhoto';
+import replaceUsers from './replaceUsers';
+
+export default function gittenify(store) {
+  const userMap = {};
```

Comment:
Just open an issue for that and label it with `improvement` or something like that :)